src/app/add-dependency/add-dependency.component.html

```diff
@@ -173,6 +172,14 @@ <h2 class="card-pf-title search-box">
                           <span id="sec">Security</span>
                         </div>
                       </div>
+                      </div>
+                      <div class="list-group-item-text">
+                        <p *ngIf="!master.showMore" class="description">{{master.description | maxLength: 10}}</p>
```

Comment:
we can use `truncate` from patterfly-ng as well.

Reply:
Yes. But as far as I can see https://github.com/patternfly/patternfly-ng/blob/aab24fd019d57e4545e35b47bcb39cabb2799d4e/src/app/pipe/truncate/truncate.pipe.ts does truncation based on the number of characters. 

I was a little worried if that might split that word in the middle. Hence made a custom pipe which does word truncation.

Let me know your thoughts.

Reply:
makes sense for word count instead character